test/studies/shootout/reverse-complement/bharshbarg/revcomp-line.good

```diff
@@ -1,3 +1,6 @@
+revcomp-line.chpl:71: In function 'readUntil':
+revcomp-line.chpl:80: warning: push_back is deprecated - please use List.extend
+revcomp-line.chpl:83: warning: push_back is deprecated - please use List.extend
```

Comment:
These two reverse-complement should probably be rewritten to use something like the ``bytes`` type instead of ``List``, so these deprecation warnings are fine for now.